### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by rexdivakar**
Added the policy update, kindly review and approve the PR.

### CODE REVIEW

**Comment by rexdivakar** [comment]
Please tag the issue and add the labels.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+currently being supported with security updates.
+
+| Version       | Supported          |
+| -------       | ------------------ |
+| v0.1.5        |      Yes           |
+| v0.1.0-alpha  | Alpha Testing      |
+
+
+## Reporting a Vulnerability
+
```
Comment (clebsonf) [inline SECURITY.md:15]: Put a contact form to make this report. While we don't think of anything better, we can use:

security@octobear.org

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading services built on top of this software.
+
+## Vulnerability Disclosure Process
+
+Confio uses the following disclosure process for the various CosmWasm-related repos:
```
Comment (clebsonf) [inline SECURITY.md:30]: I believe that here we should change our context

File: SECURITY.md (hunk 1 above)
Comment (rexdivakar) [reply SECURITY.md:15]: do u have a link to your contact info which i can share it ?

File: SECURITY.md (hunk 2 above)
Comment (rexdivakar) [reply SECURITY.md:30]: Please let me know how you would like to add the changes too ?

**Review by clebsonf — CHANGES_REQUESTED** [review]
It turned out good. Just some necessary changes.

File: SECURITY.md (hunk 2 above)
Comment (clebsonf) [reply SECURITY.md:30]: "**Confio** uses the following disclosure process for the various **CosmWasm**-related repos:"

I believe you used the SECURITY.md from CosmOsm/Confio as a base.

We just need to change this info (CosmOsm and CONFIO) to Octobear

Comment (rexdivakar) [reply SECURITY.md]: Updated !!

**Review by clebsonf — APPROVED** [review]

**Pull request merged**